tutorials/llama_index_knowledge_base_tutorial.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def display_llama_index_response(response: Response) -&gt; None:
</pre><p>Kinda takes away from the flow of the tutorial. Is there a way to have all these utility functions be defined once at the top?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>